tests/filecheck/backend/riscv/canonicalize.mlir

```diff
@@ -126,4 +129,9 @@ builtin.module {
 
 // CHECK-NEXT:   riscv.fsw %i2, %f2, 12 : (!riscv.reg<>, !riscv.freg<>) -> ()
 
+// CHECK-NEXT:   %add_lhs_rhs = riscv.li 2 : () -> !riscv.reg<>
```

Comment:
I'm confused about this change, I thought we merged it in #1566 already? I wonder why GitHub is not showing merge conflicts here.

Reply:
Can you please merge main into this PR to get rid of this change?